hpeOneView/resources/servers/enclosure_groups.py

```diff
@@ -36,9 +36,6 @@ class EnclosureGroups(Resource):
     URI = '/rest/enclosure-groups'
 
     DEFAULT_VALUES = {
-        '800': {"type": "EnclosureGroupV7"},
```

Comment:
Better to keep type for API 800 and 1000. Or else if you plan to remove all the entries, then good to remove the entire dictionary.